contracts/AgentMech.sol

```diff
@@ -244,18 +323,14 @@ contract AgentMech is ERC721Mech {
         if (account == address(0)) {
             revert ZeroAddress();
         }
-        // Decrease the total number of requests by this mech
-        mapRequestsCounts[account]--;
-        numTotalRequests--;
```

Comment:
We don't decrease the number of requests now, leave them for statistics and possible karma. The more valuable agent mech will be the one that has fewer requests than deliveries, meaning it was not asked as a priority mech a lot, but it kept delivering instead of priority mechs.